lit_gpt/config.py

```diff
@@ -17,12 +17,14 @@
 class Config:
     name: str = ""
     hf_config: dict = field(default_factory=dict)
+    scale_wte_output: bool = False
```

Comment:
People don't know what wte is

```suggestion
    scale_embedding: bool = False
```